tgui/packages/tgui/interfaces/Uplink.js

```diff
@@ -0,0 +1,188 @@
+import { Fragment } from "inferno";
+import { useBackend, useLocalState } from "../backend";
+import { Button, Box, Section, Tabs, Icon, Flex, Input } from "../components";
+import { FlexItem } from "../components/Flex";
+import { Window } from "../layouts";
+import { createSearch, decodeHtmlEntities } from 'common/string';
+import { flow } from 'common/fp';
+import { filter, sortBy } from 'common/collections';
+
+export const Uplink = (props, context) => {
+  const { act, data } = useBackend(context);
+
+  const [tabIndex, setTabIndex] = useLocalState(context, 'tabIndex', 0);
+
+  const PickTab = index => {
+    switch (index) {
+      case 0:
+        return <ItemsPage />;
+      case 1:
+        return <ExploitableInfoPage />;
+      default:
+        return "SOMETHING WENT VERY WRONG PLEASE AHELP";
+    }
+  };
+  return (
+    <Window theme="syndicate">
+      <Window.Content scrollable>
+        <Tabs>
+          <Tabs.Tab
+            key="PurchasePage"
+            selected={tabIndex === 0}
+            onClick={() => setTabIndex(0)}>
+            <Icon name="shopping-cart" />Purchase Equipment
+          </Tabs.Tab>
+          <Tabs.Tab
+            key="ExploitableInfo"
+            selected={tabIndex === 1}
+            onClick={() => setTabIndex(1)}>
+            <Icon name="user" />Exploitable Information
+          </Tabs.Tab>
+          <Tabs.Tab
+            key="LockUplink"
+            // This cant ever be selected. Its just a close button.
+            onClick={() => act('lock')}>
+            <Icon name="lock" />Lock Uplink
```

Comment:
`Tabs.Tab` has an `icon` property - no need to add the icon yourself in its contents